tests/cache/test_identity_redis_cache.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2023 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Image Redis Cache Tests."""
+
+from __future__ import absolute_import
+
+from invenio_communities.proxies import current_identities_cache
+
+
+def test_cache_deletion(app):
```

Comment:
Idea: we can take inspiration for more tests from [here](https://github.com/inveniosoftware/invenio-rdm-migrator/blob/master/tests/streams/test_cache.py) 😇

Reply:
I have improved the tests with your suggestion of testing the default timeout expiration. I think all cases are now covered unless I am missing something